package.json

```diff
@@ -81,5 +85,12 @@
     ],
     "sourceMap": false,
     "instrument": false
+  },
+  "lint-staged": {
+    "packages/*/src/**/*.js": [
+      "prettier --trailing-comma all --write",
```

Comment:
You can use https://github.com/not-an-aardvark/eslint-plugin-prettier instead here